schema.md

```diff
@@ -0,0 +1,204 @@
+# Queries
+
+All queries must have a `secret` field in the request body with the correct key. 
+
+If a field in the response is not set it will not be ommited, it will simply be empty.
```

Comment:
If for example I didn't have a location set my (simplified) JSON file might look like this if fields were omitted:
```json
{
    "User": {
        "Name": "Cathal O'Callaghan   
    }
}
```
Whereas as we have it, when a field is not set it is simply empty it would be :
```json
```
{
    "User": {
        "Name": "Cathal O'Callaghan   
        "Location": ""   
    }
}
```

In terms of parsing this, if it was omitted then the js would have to first check if the field was existing or it'd get an error if not handled right. With this the js can just can just print nothing to screen in the location text area if it's not set or text if it is